website/docs/docs/building-a-dbt-project/package-management.md

```diff
@@ -108,33 +108,80 @@ To find the latest release for a package, navigate to the `Releases` tab in the
 
 As of v0.14.0, dbt will warn you if you install a package using the `git` syntax without specifying a version (see below).
 
-#### Private packages
-Private packages can be installed by using the SSH configuration on the machine or by passing in a username and password with the git URL. It is recommended that you use SSH authentication method where possible.
+### Private packages
+
+#### SSH Key Method
+Private packages can be installed via SSH keys, which authenticate to remote servers. You'll need to add your key to your git provider and update your ~/.ssh/config file to manage your settings.
```

Comment:
```suggestion
Private packages can be installed via SSH keys, which authenticate to remote git servers. You'll need to add your key to your git provider and update your ~/.ssh/config file to manage your settings.
```

Reply:
this is still _kind of_ funny - you usually don't need to touch `~/.ssh/config` when setting up a git connection over SSH. I think we already litigated these lines and they're not wrong... I think it might just lead to some questions!

If you're unhappy with them, we can spend some more time workshopping this together. If you think it's ok, then I don't find them offensive enough to protest :)